libbpf-rs/src/map.rs

```diff
@@ -143,6 +143,28 @@ impl Map {
         self.value_size
     }
 
+    /// Size of the value for lookups in bytes. Respects the space needed when looking up values in
+    /// per-cpu maps.
+    fn lookup_value_size(&self) -> Result<usize> {
```

Comment:
Can you add a comment above `MapType` to remind editors to update this function too?

Reply:
Done. I added the comment as a normal, non-doc comment so this implementation note does not show up in the user documentation.